utils/build/docker/nodejs/app.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
-if [ ${UDS_WEBLOG:-} = "1" ]; then
-    ./set-uds-transport.sh
+if [ -e /volumes/dd-trace-js ]; then
```

Comment:
If any error in `npm link`, this will make the error easier to see in the output 

```suggestion
set -e

if [ -e /volumes/dd-trace-js ]; then
```